Tests/XCUITests/SaveLoginsTests.swift

```diff
@@ -90,15 +90,11 @@ class SaveLoginTest: BaseTestCase {
         waitForExistence(app.tables["Login List"])
         XCTAssertTrue(app.staticTexts[domain].exists)
         // XCTAssertTrue(app.staticTexts[domainSecondLogin].exists)
-        if processIsTranslatedStr() == m1Rosetta {
-            XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 2)
+        // Workaround for Bitrise specific issue. "vagrant" user is used in Bitrise.
+        if (ProcessInfo.processInfo.environment["HOME"]!).contains(String("vagrant")) {
```

Comment:
It is not ideal but let's have this for now until we find a better way to identify that the test runs on M1 / Intel, Bitrise / locally...